src/shapes/ActiveSelection.ts

```diff
@@ -146,9 +146,7 @@ export class ActiveSelection extends Group {
   }
 
   /**
-   * If returns true, deselection is cancelled.
-   * @since 2.0.0
-   * @return {Boolean} [cancel]
+   * @override remove all objects
```

Comment:
why removing the comment?

Reply:
I think the jsdoc concats the comments of the super method so there is no need, it is spam

Reply:
that is bad, because if i m just reading the codebase i can't concat anything

Reply:
And if you hover with your mouse?

Reply:
<img width="651" alt="Screenshot 2024-01-11 at 10 32 50" src="https://github.com/fabricjs/fabric.js/assets/34343793/6a515fd5-569c-4e02-a176-433c180bb81e">
 I can restore it but my point is that then you have to update a signle comment everywhere 
That is not manageable

Reply:
yeah VSCODE does this, i m just not sure how it is supposed to be by the JSDOCS logic